### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by shilpa-padgaonkar** [camaraproject/QualityOnDemand#242] [General]
As far as I can understand, if response to create session leads to qosStatus "requested", then the developer can later receive the notification, qosStatus "available". However if the create session response leads directly to qosStatus "available", the notification with status "available" will be skipped. Similarly, before explicit delete session or delete session due to timer expiry, there is no notification sent as qosStatus "unavailable".  Is this understanding correct?
If yes, what is the thought around always sending notifications whenever qosStatus becomes available or unavailable (due to whatever conditions).

**Comment by hdamker** [comment]
Got addressed within PR #258 and is included within v0.10.0
